apps/server/src/modules/task/uc/task.uc.spec.ts

```diff
@@ -103,34 +112,27 @@ describe('TaskUC', () => {
 		},
 	};
 
-	const setAuthorizationServiceMock = {
-		getPermittedCourses: (courses: Course[] = []) => {
-			const spy = authorizationService.getPermittedCourses.mockResolvedValue(courses);
+	const setCourseRepoMock = {
+		findAllForTeacher: (courses: Course[] = []) => {
+			const spy = courseRepo.findAllForTeacher.mockResolvedValue([courses, courses.length]);
 			return spy;
 		},
-		getPermittedLessons: (lessons: Lesson[] = []) => {
-			const spy = authorizationService.getPermittedLessons.mockResolvedValue(lessons);
+		findAllByUserId: (courses: Course[] = []) => {
+			const spy = courseRepo.findAllByUserId.mockResolvedValue([courses, courses.length]);
 			return spy;
 		},
-		hasTaskPermission: (hasWritePermission = false) => {
-			const spy = authorizationService.hasTaskPermission.mockReturnValue(hasWritePermission);
-			return spy;
-		},
-		hasOneOfTaskDashboardPermissions: () => {
-			const spy = authorizationService.hasOneOfTaskDashboardPermissions.mockImplementation(
-				(_: User, permission: TaskDashBoardPermission | TaskDashBoardPermission[]) => {
-					const hasPermission: boolean = Array.isArray(permission)
-						? permission.some((p) => currentPermissions.includes(p))
-						: currentPermissions.includes(permission);
-
-					return hasPermission;
-				}
-			);
-
+	};
+	const setLessonRepoMock = {
+		findAllForTeacher: (lessons: Lesson[] = []) => {
+			const spy = lessonRepo.findAllByCourseIds.mockResolvedValue([lessons, lessons.length]);
 			return spy;
 		},
 	};
 
+	/* 	courseRepo.findAllByUserId.mockResolvedValue([[course], 1]);
```

Comment:
comments can be removed, also in line 150, line 159, line 226, line 237, line 461.